.gitignore

```diff
@@ -1,4 +1,4 @@
 .terraform
-terraform.tfstate
```

Comment:
Revert these in gitignore.

Reply:
I'm not sure when `terraform.tfstate` (tests should confine these to an already ignored dir) or `terraform.tfvars`
 (all vars are packaged, why would we need override) would exist in the repo. Can you explain the use case for either of those existing within the module?

Reply:
During work, I run code from `examples` on my mac and don't want to commit `terraform.tfstate` file. Sometimes I also have special `terraform.tfvars` in specific examples but it is rather seldom when I need it.

Reply:
Removed, but consider moving the examples from manually run to kitchen terraform also. Even a test fixture without tests is valuable and lowers the developer effort. That would make the gitignores here irrelevant.